functions/package.json

```diff
@@ -13,9 +13,12 @@
   },
   "main": "index.js",
   "dependencies": {
+    "@google-cloud/local-auth": "^1.0.1",
     "firebase-admin": "^9.2.0",
     "firebase-functions": "^3.11.0",
     "glob": "^7.1.6",
+    "googleapis": "^68.0.0",
+    "nodemailer": "^6.5.0",
```

Comment:
this is unused?

Reply:
googleapis is needed for the email sending but I removed the nodemailer